template/lib.typ

```diff
@@ -227,27 +227,151 @@ text)
   }
 
 
+#let pretty-header(numeration: "1", main: false) = {
+  if(main){
+      context {
+    // Header for the main content
+    let titles = query(selector(heading.where(level: 1).or(heading.where(level:2))))
+
+    let title_left
+    let title_right
+    let chapter 
+
+    let page = counter(page).get().first()
+    
+    for title in titles {
+    let loc = title.location() 
+    let title_numbering = if title.numbering == none {
+          none
+        } else {
+          context numbering("1.", ..counter(heading).at(loc))
+        } 
+
+        // Chapter numbering for the left headers
+        if(title.level == 1){
+         chapter = [Capítulo ] + [#numbering("1",..counter(heading).at(loc))] + [. ] 
+        } else {
+         none
+        }
+
+        // Left headers for the main content
+        if locate(loc).page() <= here().page() and title.level == 1 {
+          title_left = strong([#numbering(numeration,page)] + h(1.4em) + chapter + title.body);
+        }
+
+        // Right headers for the main content
+        if locate(loc).page() <= here().page() and title.level == 2 {
+          title_right = strong([#title_numbering ]+ title.body + h(1.4em) + [#numbering(numeration,page)]);
+        }
+
+        // Skip the first header of a section
+        if(title.level == 1 and locate(loc).page() == here().page()){
+          title_left = none;
+        }
+    }
+
+    // Header placement
+    if title_left != none{
+    if calc.odd(page){
+
+    set text(1em)  
```

Comment:
Does this set do anything? AFAIK, ems are sizes relative to the current font size, so setting the size to 1em is like setting it to 100% (same value as before)